sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -401,6 +401,92 @@ case class DayOfYear(child: Expression) extends UnaryExpression with ImplicitCas
   }
 }
 
+@ExpressionDescription(
+  usage = "_FUNC_(date) - Returns timestamp from seconds.",
```

Comment:
`_FUNC_(seconds) - Creates timestamp from the number of seconds since UTC epoch.`